index.html

```diff
@@ -6,46 +6,150 @@
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
 </head>
 <body>
 <header>
     <div class="container topbar">
-        <div class="logo">
-            <a href="index.html"><img src="images/logo.png" alt="logo"></a>
+        <div class="social">
+            <ul id="desksocial">
+                <li class="socialmenu"><a href="https://www.facebook.com/"><i class="fab fa-facebook-f"></i></a></li>
+                <li class="socialmenu"><a href="https://twitter.com/"><i class="fab fa-twitter"></i></a></li>
+                <li class="socialmenu">English</li>
+                <li class="socialmenu">My Page</li>
+                <li class="socialmenu">Logout</li>
+            </ul>
         </div>
+
+    <nav id="navbar">
         <div class="menu">
-            <i class="fa fa-bars"></i>
-            <ul id="menu-items">
-                <li><a href="index.html">Home</a></li>
-                <li><a href="about.html">About us</a></li>
-            </ul>
+            <i id="openModal" class="fa fa-bars"></i>
+            <div id="myModal" class="modal">
+                <i id="close" class="fa fa-times hidden"></i>
+                <ul id="menu-items" class="hidden">
+                    <li>
+                        <a href="index.html">
+                        <img id="logotobar" src="./Assets/Artificial Inteligence Global Summit 2023.png">
+                        </a>
+                    </li>
+                    <li id="home"><a class="alink" href="index.html">Home</a></li>
+                    <li><a class="alink" href="about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li id="redbox">AI Campaign</li>
+                </ul>
+            </div>
         </div>
     </div> 
+    </nav>
 </header>
 
 <main>
 
+
 <section id="home1">
  <div class="container cont1">
-    <h3>"Hello! Sharing world"</h3>
-    <h1>CREATIVE COMMONS GLOBAL SUMMIT 2023</h1>
-    <div class="textbox">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</div>
-    <h2>2015.10.15(THU)~16(FRI)</h2>
-    <p>@ National Museam of Karea, Art Center Nabi and more.." </p>
+    <div id="textcontainerhome1">
+    <h3 id="spacingtop">"AI, friend or foe?"</h3>
+    <h1 id="h1desk">ARTIFICIAL INTELLIGENCE GLOBAL SUMMIT 2023</h1>
+    <div class="textbox">How should we plan ahead for the disruption that AI will bring to our world? Delegates from over 20 countries will be sharing their expertise with us. We look forward to hosting you in Cape Town, South Africa for this unique event.</div>
+    <h2 class="contdesk">2023.10.15(THU) ~ 16(FRI)</h2>
```

Comment:
- [ ] This text is too large compared to that in the original design. Please scale it down and add space between in and the next section as seen in the original design
### Your Implementation
![image](https://github.com/dvcorreia1984/Capstone-Project/assets/45460192/62ebe310-b8ef-4020-b726-0a39c0c116fe)
### Required
![image](https://github.com/dvcorreia1984/Capstone-Project/assets/45460192/e5f3ee5e-9818-4e6d-b346-0613bb6a01ed)

Reply:
Ok, I think I managed to fix this.